solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -221,6 +221,9 @@ contract RandomBeacon is Ownable {
         relay.setRelayEntrySubmissionEligibilityDelay(10);
         relay.setRelayEntryHardTimeout(5760); // ~24h assuming 15s block time
         relay.setRelayEntrySubmissionFailureSlashingAmount(1000e18);
+
+        groups.setRelayEntryTimeout(relay.relayEntryTimeout());
+        groups.setGroupLifetime(groupLifetime);
```

Comment:
We no longer need `groupLifetime` to be a field of `RandomBeacon` and `updateGroupCreationParameters` governance function no longer works. We need similar approach as for `updateRelayEntryParameters` and keep `groupLifetime` only in `Groups`.

Reply:
Moved to Groups.sol fd01258